nwbinspector/config.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "object",
```

Comment:
@bendichter Should we version this schema?

Reply:
good call!